sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala

```diff
@@ -1750,11 +1750,10 @@ private[sql] object QueryExecutionErrors extends QueryErrorsBase with ExecutionE
   }
 
   def invalidStreamingOutputModeError(
```

Comment:
Currently, there is no way to add test cases related to this error class.

Because in the two places(`StateStoreSaveExec` and `SessionWindowStateStoreSaveExec`) where exceptions are thrown, output mode param is already of `OutputMode` type, and an empty assert check is added to the run method.